CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Braintree Android SDK Release Notes
 
+## Unreleased
+
+* PayPalNativeCheckout
+  * Bumping native-checkout version to 0.8.1
+  * Bugfix to support logging out when client id's switch
+  * Bugfix for auth-sdk obfuscation
```

Comment:
What specifically is being fixes with the obfuscation of the auth SDK? Is this entry needed for public facing merchants?

Reply:
it was a duplicate class file, its actually probably not needed to declare here. We saw an issue with LMVH about it and something broke with obfuscation but i'll remove this entry